### DISCUSSION TITLE
[Question][Hw4] Submission is unavailable

### DISCUSSION THREAD

**Discussion by ehdkacjswo** [prosyslab-classroom/cs524-program-analysis#302] [Q&A]
Name: Doam Lee

I'm trying to submit my homework, but turnitin is on maintenence now. What should I do?

![Screenshot_20240420_153114_Chrome](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/25629264/4b8f0e10-88fe-4a54-9655-96776566bd79)

Time is differen because I'm currently in Lisbon

**Comment by goodtaeeun** [comment]
This is a direct link to the Gradescope course page. https://www.gradescope.com/courses/732230
Please try if this works for you.

**Comment by ehdkacjswo** [comment]
I've tried the link but there is no KAIST on school credential and I can't login with my other account too.

**Comment by goodtaeeun** [comment]
If you have finalized the implementation of your homework, I can submit your repository for you.
Would that be okay?

**Comment by ehdkacjswo** [comment]
If you could, yes please. Thanks you!

**Comment by goodtaeeun** [comment]
I have just uploaded your work.
